calebwagner/button-the-clown
# create reservation field set

I create the reservation form and invoked the ServiceForm function that generates the field sets in the ClownService.js module.

Steps to check code:
1. git fetch --all
2. git checkout cjw-request-form 
3. code. 
4. check for field set elements in ReservationForm.js module
5. checkout to make sure the function in properly invoked
